packages/@react-spectrum/tooltip/stories/TooltipTrigger.stories.tsx

```diff
@@ -176,6 +176,24 @@ export const TooltripTriggerInsideActionGroup: TooltipTriggerStory = {
   )
 };
 
+const TooltipDivRender = (args) => {
```

Comment:
bit weird, this is a render function, not a component, could cause problems if storybook changes how their internals work

instead, make this a component taking props (looks exactly the same, just change args to props)
and then down below for render you should call `(args) => <TooltipDivRender {...args} />`

Reply:
makes sense. yeah i'm still getting used to this style of stories, wasn't getting it to work how i'd expected